payment_lyra/helpers/constants.py

```diff
@@ -7,108 +7,205 @@
+    "BIZZBEE_CDX_SB": "BizzBee gift card - Sandbox mode",
+    "BOULANGER": "Boulanger brand card",
+    "BOULANGER_SB": "Boulanger brand card - Sandbox mode",
+    "BRICE_CDX": "Brice gift card",
+    "BRICE_CDX_SB": "Brice gift card - Sandbox mode",
+    "CB": "CB",
+    "CHQ_DEJ": "Chèque Déjeuner electronic meal voucher",
+    "CONECS": "Conecs electronic meal voucher",
+    "CORA": 'CORA" Aurore card',
+    "CORA_BLANCHE": 'CORA Blanche" Aurore card',
+    "CORA_PREM": 'CORA Premium" Aurore card',
+    "CORA_VISA": 'CORA VISA" Aurore card',
+    "CVCO": "Chèque-Vacances Connect",
+    "DINERS": "Diners",
+    "DISCOVER": "Discover",
+    "E_CV": "e-Chèque-Vacances",
+    "E-CARTEBLEUE": "e-carte bleue",
+    "ECCARD": "Euro-Cheque card",
+    "EDENRED_EC": "Edenred Ticket Eco Chèque",
+    "EDENRED_SC": "Sports & Culture Edenred Ticket",
+    "EDENRED_TC": "Edenred Ticket Compliment",
+    "EDENRED_TR": "Edenred meal voucher",
+    "FRANFINANCE_3X": "Franfinance payment in 3X",
+    "FRANFINANCE_4X": "Franfinance payment in 4X",
+    "FULLCB3X": "Payment in 3 installments with no fees with BNPP PF",
+    "FULLCB4X": "Payment in 4 installments with no fees with BNPP PF",
+    "GIROPAY": "Giropay",
+    "GOOGLEPAY": "Google Pay wallet payment",
+    "IDEAL": "iDeal Internet Banking",
+    "ILLICADO": "Illicado Gift Card",
+    "ILLICADO_SB": "JouéClub gift card - Sandbox mode",
+    "JCB": "JCB",
+    "JOUECLUB_CDX": "Jouéclub gift card",
+    "JOUECLUB_CDX_SB": "JouéClub gift card - Sandbox mode",
+    "JULES_CDX": "Jules gift card",
+    "JULES_CDX_SB": "Jules gift card - Sandbox mode",
+    "KLARNA": "Klarna Internet Banking",
+    "LEROY-MERLIN": "Leroy-Merlin brand card",
+    "LEROY-MERLIN_SB": "Leroy-Merlin brand card - Sandbox mode",
+    "MAESTRO": "Maestro",
+    "MASTERCARD": "Mastercard",
+    "MASTERPASS": "MasterPass",
+    "MULTIBANCO": "Multibanco",
+    "MYBANK": "MyBank",
+    "NORAUTO": "Norauto brand card",
+    "NORAUTO_SB": "Norauto brand card - Sandbox mode",
+    "ONEY_3X_4X": "Oney 3x 4x payment",
+    "ONEY_ENSEIGNE": "Oney partner Brand Cards.",
+    "PAYDIREKT": "PayDirekt",
+    "PAYLIB": "Wallet Paylib",
+    "PAYPAL": "PayPal",
+    "PAYPAL_SB": "PayPal - Sandbox mode",
+    "PICWIC": "PicWic brand card",
+    "PICWIC_SB": "PicWic brand card - Sandbox mode",
+    "POSTFINANCE": "PostFinance",
+    "POSTFINANCE_EFIN": "PostFinance E-finance",
+    "PRESTO": "Presto by Cetelem online credit solution",
+    "PRZELEWY24": "Przelewy24",
+    "SCT": "SEPA CREDIT TRANSFER",
+    "SDD": "SEPA DIRECT DEBIT",
+    "SODEXO": "Sodexo electronic meal voucher",
+    "SOFORT_BANKING": "Sofort Banking",
+    "UNION_PAY": "Union Pay",
+    "VILLAVERDE": "Villaverde gift card",
+    "VILLAVERDE_SB": "Villaverde gift card - Sandbox mode",
+    "VISA": "Visa",
+    "VISA_ELECTRON": "Visa Electron",
+    "VPAY": "Vpay",
+    "WECHAT": "WeChat",
 }
 
+# https://payzen.io/en-EN/form-payment/reference/vads-currency.html
```

Comment:
this comments: #15 (comment) make me realized that it should use those data https://docs.lyra.com/fr/collect/form-payment/standard-payment/vads-currency.html instead